### DISCUSSION TITLE
能否添加合集/播放列表功能

### DISCUSSION THREAD

**Discussion by ZHAY10086** [ywmoyue/biliuwp-lite#37] [feature]
如题，网页版虽然有合集/播放列表功能，但是随着连续播放的视频越来越多会逐渐变卡。
不知道UPW能不能也实现这个功能，避免网页版卡顿。

**Comment by ywmoyue** [comment]
暂时可以先在其他平台订阅合集，再在app中的收藏夹中找到订阅的合集，即可以播放列表的方式播放合集视频
